src/frontend/typer/sanity/modaler/pensjonsperiode.ts

```diff
@@ -0,0 +1,23 @@
+import { LocaleRecordBlock } from '../../common';
+
+export interface IPensjonsperiodeTekstinnhold {
+    dato: {
+        farStartdato: { feilmelding: LocaleRecordBlock; sporsmal: LocaleRecordBlock };
+        fikkStartdato: { feilmelding: LocaleRecordBlock; sporsmal: LocaleRecordBlock };
+        narAvsluttetDato: { feilmelding: LocaleRecordBlock; sporsmal: LocaleRecordBlock };
+    };
+    farPensjonNa: {
+        feilmelding: LocaleRecordBlock;
+        sporsmal: LocaleRecordBlock;
+    };
+    flerePerioderSporsmal: LocaleRecordBlock;
```

Comment:
Tenkte på at vi kanskje kunne laget et felles interface for det som er felles for modalene? typ de feltene som alle modalene importerer. Har jo lagt til et nytt felt i den felles malen nå, så hadde vært litt digg å bare legge den inn ett sted.

Reply:
Ja, sant, kan endre det 😊